### PR TITLE
Add support for filesystem_type parameter for volume resource

### PR DESCRIPTION
**Pull request by pgrzesik**
Added support for filesystem_type flag for volumes, related to issue: https://github.com/terraform-providers/terraform-provider-digitalocean/issues/103

Required upgrading godo dependency to version that supports this flag.

### CODE REVIEW

File: digitalocean/resource_digitalocean_volume.go [hunk 1]
```diff
@@ -53,6 +53,12 @@ func resourceDigitalOceanVolume() *schema.Resource {
 				Optional: true,
 				ForceNew: true, // Update-ability Coming Soon ™
 			},
+
+			"filesystem_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true, // Update-ability Coming Soon ™
```
Comment (andrewsomething) [inline digitalocean/resource_digitalocean_volume.go]: Let's remove this comment. The DigitalOcean API does not currently offer a way to re-format a volume, so "update-ability" isn't something we can promise.

**Comment by andrewsomething** [comment]
Hi @pgrzesik,

Thanks so much for the contribution! Besides one minor nitpick, everything looks great. Let me know if you don't have the bandwidth right now, and I can make that change when I merge it in.

Thanks!

**Comment by pgrzesik** [comment]
hey @andrewsomething, that's a great news! I just pushed a commit that removes this comment (bad on my part, sorry about that!), so I think it's now ready to merge :) 

That was fun, thanks a lot!

**Comment by eddiezane** [comment]
@pgrzesik thanks for your contribution! Can you please shoot me an email to eddiezane [at] digitalocean [dot] com when you get a chance?

**Pull request merged**